src/dmd/root/utilities.d

```diff
@@ -0,0 +1,51 @@
+/***
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Various utilities.
+ *
+ * Copyright:   Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/root/utilities.d, root/_utilities.d)
+ * Documentation: https://dlang.org/phobos/dmd_root_utilities.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/root/utilities.d
+ */
+module dmd.root.utilities;
+
+/// Resets all static fields of the given aggregate to their `init` values.
+void resetStaticFields(T)()
+    if (is(T == class) || is(T == struct))
```

Comment:
D Style is to have the constraint on the same indentation

Reply:
Fixed.